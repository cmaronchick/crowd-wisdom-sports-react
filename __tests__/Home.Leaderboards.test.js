import React from 'react';
import HomeLeaderboards from '../src/components/Home.Leaderboards'
import renderer from 'react-test-renderer'

describe('it should present the Home Leaderboards', () => {
    
    test('present a blank leaderboard when there is no data', () => {
        const homeleaderboardsrender = renderer.create(<HomeLeaderboards />).toJSON();
        console.log({homeleaderboardsrender});
        expect(homeleaderboardsrender).toMatchSnapshot();
    })

})
