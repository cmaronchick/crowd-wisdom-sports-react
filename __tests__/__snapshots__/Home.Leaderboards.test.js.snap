// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should present the Home Leaderboards present a blank leaderboard when there is no data 1`] = `
<div>
  Loading Leaderboard
</div>
`;
